README.md

```diff
@@ -42,19 +42,32 @@ on first run, this will be populated with a basic config if it doesn't exist.
 The default config is:
 
 ```toml
-# How often to check battery status, in seconds.
-interval_secs = 30
+# How often to check battery status, in milliseconds.
+interval = 30000
 
-# At what percentage of battery capacity to notify about low battery.
+# At what percentage of battery capacity to notify about low battery, set to 0 to disable.
 low_pct = 40
 
-# At what percentage of battery capacity to notify and run sleep_command.
+# At what percentage of battery capacity to notify and run sleep_command, set to 0 to disable.
 sleep_pct = 15
 
+# Custom commands to run on battery state change.
+#
+# These commands and the sleep_command that follows are run like this: 
+# $SHELL -c <your-command>
+#
+# Using "true" like this is a no-op for most shells and the default.
+battery_state_charging_command = "true"
+battery_state_discharging_command = "true"
```

Comment:
Since this is now getting a lot of values, let's update to:

Use sections, then you can have thing like:

```
[state_notif]
full_cmd = ...
...
```

...and not have to prefix everything